chapter03/qli/src/main/java/spring/training/model/Account.java

```diff
@@ -0,0 +1,111 @@
+package spring.training.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Account {
+	private int accountNumber;
+	private String accountDescription;
+	private double accountAmount;
+	private Customer customer;
+	
+	public final static double MAX_AMOUNT = 100000000;
+
+	private List<AccountTranscation> transactions = new ArrayList<>();
+	
+	
+	public int getAccountNumber() {
+		return accountNumber;
+	}
+
+	public Account setAccountNumber(int accountNumber) {
+		this.accountNumber = accountNumber;
+		return this;
```

Comment:
Returning `this` from a setter? This is for creating the right object in a line for `BankBeanConfig`?